package.json

```diff
@@ -117,15 +111,24 @@
     "@types/enzyme": "3.1.15",
     "@types/enzyme-adapter-react-16": "1.0.3",
     "@types/jest": "26.0.4",
+    "@typescript-eslint/eslint-plugin": "^2.18.0",
+    "@typescript-eslint/parser": "^2.18.0",
     "@xstate/test": "0.3.0",
     "@yarnpkg/lockfile": "1.1.0",
     "enzyme": "3.8.0",
     "enzyme-adapter-react-16": "1.7.1",
     "enzyme-to-json": "3.3.5",
+    "eslint": "5.6.0",
+    "eslint-plugin-jsdoc": "^21.0.0",
+    "eslint-plugin-markdown": "^1.0.1",
+    "eslint-plugin-patternfly-react": "^4.1.2",
+    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-react": "^7.18.0",
     "husky": "0.14.3",
     "jest": "26.1.0",
     "jest-emotion": "10.0.32",
     "lint-staged": "7.2.0",
```

Comment:
Looks like we can remove the `lint-staged` dependency as well

Reply:
done